### DISCUSSION TITLE
Is there a way to execute static window rules dynamically?

### DISCUSSION THREAD

**Discussion by titaniumtraveler** [hyprwm/Hyprland#4853] [Q&A] [answered]
Specifically I want windows that changes from `fullscreen:0` to `fullscreen:1` to be in `fakefullscreen`.
For example when pressing `f` in a youtube video it should always be `fakefullscreen` and not fullscreen, regardless what the window state was prior to that. \
(I'm not 100% sure that window rules are the correct tool to achieve what I want, so if there is another way to achieve what I want, I would be open to that.)

**Comment by vaxerski** [comment, marked as answer]
no

**Reply by nativerv** [reply]
is there any reason to disallow that or you just haven't implemented it?
my usecase is `float`-ing xdg-desktop-portal-termfilechooser in which i can only set the title with escape codes, which does it immediately but still *after* the window is created, and the rule does not work for it because of that
i also think that a lot of stuff may work this way (not creating the window with the correct title but setting it later during initialization), or else i don't see why my `move` rule don't work with telegram floating player window but `noborder` does.

**Reply by vaxerski** [reply]
reason is "not implemented"
